dapp/src/components/Layout.tsx

```diff
@@ -1,27 +1,43 @@
 import React from "react"
 import { Outlet } from "react-router-dom"
 import { Flex } from "@chakra-ui/react"
-import { useMobileMode } from "#/hooks"
+import { useIsEmbed, useMobileMode } from "#/hooks"
 import DocsDrawer from "./DocsDrawer"
 import Header from "./Header"
 import ModalRoot from "./ModalRoot"
 import Sidebar from "./Sidebar"
+import MobileModeBanner from "./MobileModeBanner"
+
+const PADDING = "2.5rem" // 40px
+const PAGE_MAX_WIDTH = {
+  standalone: "63rem", // 1008px
+  "ledger-live": "63rem", // 1008px
+}
 
 function Layout() {
   const isMobileMode = useMobileMode()
+  const { isEmbed } = useIsEmbed()
+
+  if (isMobileMode) return <MobileModeBanner forceOpen />
+
+  const maxWidth = isEmbed
+    ? // TODO: Select correct max-width by `embeddedApp`
+      PAGE_MAX_WIDTH["ledger-live"]
```

Comment:
Can we address this in this PR, or do we want to create another ticket to track it?

Reply:
We can't do it now because we need changes that will come in https://github.com/thesis/acre/pull/761. I'll take care of it in https://github.com/thesis/acre/issues/772